### ISSUE TITLE
HN example: infinite scroll out of memory error

### ISSUE THREAD

**Issue by naartjie** [briskml/brisk#115]
Steps to reproduce:

- `esy run` the [`hacker-news`](./examples/hacker-news) example
- scroll down & wait for more items to load
- repeat until crash:

```
HackerNews(18909,0x10a73f5c0) malloc: can't allocate region
*** mach_vm_map(size=7093169413120000) failed (error code=3)
HackerNews(18909,0x10a73f5c0) malloc: *** set a breakpoint in malloc_error_break to debug
Fatal error: out of memory.
```

**Comment by wokalski** [comment]
@naartjie the first step when debugging this would be profiling this with the allocations instrument in Instruments.app

**Comment by naartjie** [comment]
Thanks @wokalski. I'll try fiddling with it, and see if I am able to get more info.

**Comment by wokalski** [comment]
It'd be interesting to see how it works with the new version of brisk-reconciler:
https://github.com/briskml/brisk-reconciler/pull/33

**Comment by naartjie** [comment]
@wokalski yup, was thinking the same. I'm keen to check it out. Hopefully this week sometime 🤞

**Comment by naartjie** [comment]
@wokalski I did a quick check, and I'm still getting the same crash with the latest reconciler. I'll try again with the profiler.
